src/controllers/organizations.js

```diff
@@ -22,9 +23,12 @@ async function updateOrgProfile(req, res) {
             'city',
             'websiteUrl',
         ]);
-        if (newOrga.email === orga.email) {
+
+        const usedEmail = await BaseUser.findOne({ email: newOrga.email });
```

Comment:
Sorry for being annoying but `usedEmail`? hshshs
`existingUser` maybe?

Reply:
sure!